src/lib/selection.js

```diff
@@ -364,20 +364,7 @@ module.exports = {
     },
 
     singleSelect: function() {
-        var isCheckboxOnlyRowSelections = this.properties.checkboxOnlyRowSelections,
-            isSingleRowSelectionMode = this.isSingleRowSelectionMode(),
-            hasCTRL = false,
-            result;
-
-        if (this.mouseDownState) {
-            //triggered programmatically
-            hasCTRL = this.mouseDownState.primitiveEvent.detail.primitiveEvent.ctrlKey;
-        }
-
-        result = (
-            isCheckboxOnlyRowSelections && isSingleRowSelectionMode ||
-            !isCheckboxOnlyRowSelections && (!hasCTRL || isSingleRowSelectionMode)
-        );
+        var result = this.properties.singleRowSelectionMode;
```

Comment:
Just not needed here. Due to naming of `checkboxOnlyRowSelections` property, I was conflating functionality. They never seemed to "get" nor care about CTRL anyway. It does still function where needed (in other parts of code).